roboteq_port: /dev/roboteq_driver
